### PR TITLE
HTML/CSS & JavaScript capstone project - Travelly Page

### PR DESCRIPTION
**Pull request by Ali-0111**
###  Learning objectives

- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements in the page.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.
- Communicate technical concepts to other technical people.

### Project requirements:
- You should personalize the content of your page. Choose a topic that is different than the one in the original design. 
- You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
- Colors.
 - Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the 
 one used in the original design).
- Layout: composition and space between elements.
 - The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
 - You can use a CSS framework (for example Bootstrap) for styling, if you want to.
- You should build only these 2 pages:
 - The home page.
 - The about page.
 - Each of these pages should have versions for 2 different screen sizes:
 - Mobile: up to 768px wide.
 - Desktop: 768px or wider.
 - Interactions
- Links
  - The home page should have a link in the menu to the about page.
  - The logo in the header links to the home page.
  - Mobile menu
  - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
  - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design 
  (colors, typography, spacings, etc.).
  - The mobile menu should have a close (X) button that closes the menu.
  - Dynamic page
  - The section "Featured speakers" should be created dynamically in JavaScript.
  - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for 
  this section dynamically.
  - Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).
  - The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give 
  appropriate credit to the author. Therefore, you must do it in the README of your project.
- Project documentation
  - Once you have finished the development of the project, you should record a video presenting the features of the project you 
  built. It is a video with a maximum length of 5 minutes. The content of the video should include:
  - A description of the project.
  - A demo of the project features (different pages and different screen sizes).
  - You should also highlight some interesting piece of code or something you built that you are very proud of.

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @Ali-0111, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has linter errors that need to be resolved. Kindly make all the linters pass and request for review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+    font-size: 7rem;
+  }
+
+  .details {
+    width: 60%;
+  }
+
+  .about {
+    padding-top: 4rem;
+  }
```
Comment (1GORDON) [inline style.css:373]: Great job @Ali-0111 🤝

- [ ] As stated in the project requirements, kindly follow the design guidelines as much as possible. You may change your main programs section to replicate the one given in the design guidelines.
- Your design
Your-Main-Program
- Given Design
Given-Main-Program

File: pages/about.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                <span></span>
+                <span></span>
+            </button>
+        </header>
+        <div class="text text-center">
+            <h3 class="py-2 my-2 ">"Hello! Sharing world"</h3>
+            <h1 class="fw-semibold normal p-1 my-2 mb-4">CREATIVE COMMONS <br>
+                GLOBAL SUMMIT 2015
+            </h1>
+            <p class="border text-center w-100 bg-white text-black my-3 py-4 px-2 fs-2">
```
Comment (1GORDON) [inline pages/about.html:29]: Great job @Ali-0111 🤝

- [ ] Your paragraphs on the mobile view of the about page seem to be distorted and they are not replicating the given design properly. You may kindly fix all the text and fonts of the About section of your mobile version and resubmit for review.
- Your design
Your-About
- Given Design
Given-About

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Ali-0111 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [x] Great job illustrating the project features in the video presentation. However, you forgot to showcase a piece of code you are most proud of or excited about building. Kindly record another video and showcase a piece of code you are excited about building.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -103,7 +103,7 @@
 <!------------------------Pleas update the link -------------->
 
 - [Project:Capstone_1](https://capstone1safari.netlify.app)
-- [Short_Presentation](https://www.loom.com/share/f3c9f2f77a374cadb4fd460c92abf808)
+- [Short_Presentation](https://www.loom.com/share/f844acdc411a484ca652a92f8dccb962)
```
Comment (Ali-0111) [inline README.md:106]: I added new link for the presentation video.

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @Ali-0111, 

Your Capstone Project is complete! Congratulations 👏 

It's time to merge it :shipit: 

## To Highlight
✔️ The repo looks professional
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Wonderful UI design and good choice of theme🥇 
✔️ You used JavaScript to display dynamically the list of Participants 
✔️ I really like languages select feature in the Top 😄 


## Optional suggestions
- Your Home or Index.html page on mobile design does not contain any footer. You may be thinking that the design does not contain the footer. The reason was that the design was meant to show the `MORE` options on the index.html file's footer. But it must contain a footer. By the way, didn't you think that, "why this page is missing the footer in the mobile version"? So spare some time to add a footer in `index.html` mobile version.
The requirement of that footer is  
```
It contains a logo aligned to the left and text aligned to the right.
```
A reference from my Capstone project:
image

- When you click on `MORE` button. The speaker list would expand, and the `MORE` button would disappear. What do you think, shouldn't it be like that?
image
- Overall your project is great. Implement the above OPTIONAL suggestions. Since you would be displaying this Capstone project everywhere. The above suggestions would help to make it better. Best of luck for the next modules 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**